Xwt.XamMac/Xwt.Mac/ViewBackend.cs

```diff
@@ -883,5 +909,60 @@ public void UpdateChildPlacement ()
 			child.Frame = new CGRect ((nfloat)cx, (nfloat)cy, (nfloat)cwidth, (nfloat)cheight);
 		}
 	}
+
+	class DraggingSource : NSDraggingSource
```

Comment:
```suggestion
	class DraggingSource : INSDraggingSource
```

Reply:
Fixed (now it's `class DraggingSource : NSObject, INSDraggingSource`.  I believe that's correct (and it seems to work) but feel free to confirm.